src/main/java/frc/robot/utils/controllerUtils/MultiButton.java

```diff
@@ -15,25 +18,30 @@ public class MultiButton {
     private final BooleanSupplier trigger;
     private final byte buttonID;
     private BiConsumer<Boolean, Boolean>[] buttonActions;
-    private boolean pressed;
+    private boolean pressed = false;
     private boolean pressedLast = false;
     private int layerCount;
     private int buttonLayer = 0;
+    private final ButtonHelper parent;
+    private final Command command;
 
     @SuppressWarnings("unchecked")
-    public MultiButton(BooleanSupplier trigger, byte buttonID, int layer, Command command, RunCondition runCondition) {
+    public MultiButton(BooleanSupplier trigger, byte buttonID, int layer, Command command, RunCondition runCondition, ButtonHelper parent) {
         this.trigger = trigger;
         this.buttonID = buttonID;
         this.layerCount = layer + 1;
+        this.parent = parent;
+        this.command = command;
         buttonActions = new BiConsumer[layerCount];
 
         CommandScheduler.getInstance().getDefaultButtonLoop().bind(this::updateButton);
 
         addLayer(layer, command, runCondition);
+        updateNetworkTables();
     }
 
     public MultiButton(BooleanSupplier trigger, byte buttonID) {
-        this(trigger, buttonID, 0, new InstantCommand(() -> {}), RunCondition.WHEN_PRESSED);
+        this(trigger, buttonID, 0, new InstantCommand(() -> {}), RunCondition.WHEN_PRESSED, null);
```

Comment:
This needs to have the ButtonHelper parent as well. Otherwise, any buttons that are instantiated without commands won't interact with the dashboard correctly.